src/main/java/uk/gov/justice/hmpps/prison/service/BookingService.java

```diff
@@ -565,18 +565,14 @@ private void validateScheduledEventsRequest(final LocalDate fromDate, final Loca
      */
     public void verifyBookingAccess(final Long bookingId, boolean accessDeniedError, final String... rolesAllowed) {
 
+        checkBookingExists(bookingId);
```

Comment:
Bomb out straight away if no booking exists - this also does the check for null bookingId